src/schema/v2/search/SearchResolver.ts

```diff
@@ -94,6 +94,15 @@ export class SearchResolver {
     }
   }
 
+  isForEigenVersionRequiringWorkaround() {
+    const { userAgent } = this.context
+    return (
+      userAgent &&
+      (userAgent.includes("Artsy-Mobile/6.2.0") ||
+        userAgent.includes("Artsy-Mobile/6.2.1"))
```

Comment:
This might benefit from a comment or ideally a time horizon after which the work-around can be removed.

Reply:
Yeah, I was thinking of adding [another time-bomb spec](https://github.com/artsy/metaphysics/blob/826d125963060e20f24ee8ab7939f9cc967b5e21/src/schema/v1/__tests__/partner_show_events.test.ts#L5-L10), but in this instance I think we will likely  close this out in favor of https://github.com/artsy/metaphysics/pull/2160.